samples/spring/pet-store/pom.xml

```diff
@@ -24,9 +24,8 @@
     </licenses>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <spring.version>5.3.26</spring.version>
+        <java.version>17</java.version>
```

Comment:
`java.version` is a Spring Boot specific property, isn't it? Afaik it only has an effect if the pom extends `spring-boot-starter-parent`.

Reply:
You're right. Thanks for pointing that out.